### PR TITLE
CheckV - prophages fixes and post processing gff creation step

### PR DESCRIPTION
**Pull request by mberacochea**
This PR includes:
- A fix for prophages and chechV
- GFF generation step

### CODE REVIEW

File: nextflow/configs/container.config [hunk 1]
```diff
@@ -22,7 +22,7 @@ process {
 
     withLabel: plot_contig_map {container = 'microbiomeinformatics/emg-viral-pipeline-plot-contig-map:v1' }
     withLabel: virfinder {      container = 'microbiomeinformatics/virfinder:v1.1__eb8032e' }
-    withLabel: checkV {         container = 'microbiomeinformatics/checkv:v0.8.1' }
+    withLabel: checkV {         container = 'microbiomeinformatics/checkv:v0.8.1__1' }
```
Comment (hoelzer) [inline nextflow/configs/container.config:25]: just a recommendation: for such cases you could also, in general, add an md5 checksum to the container name. E.g. the first 5 characters of the md5 sum of the `Dockerfile`.

File: docker/checkv/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@ LABEL software.version="0.8.1"
 ENV VERSION 0.8.1
```
Comment (hoelzer) [inline docker/checkv/Dockerfile:12]: we could also bump all tool versions at some point. E.g. `checkv` is at 1.0.1 already. Not necessarily now, but maybe in another update

**Review by hoelzer — APPROVED** [review]
Looks all good! I also did some test run! From my side +1 for merging. Will also check the PR #81 now

**Comment by mberacochea** [comment]
This also closes #77

File: nextflow/configs/container.config (hunk 1 above)
Comment (mberacochea) [reply nextflow/configs/container.config:25]: Captured in https://github.com/EBI-Metagenomics/emg-viral-pipeline/issues/85

File: docker/checkv/Dockerfile (hunk 2 above)
Comment (mberacochea) [reply docker/checkv/Dockerfile:12]: Captured in https://github.com/EBI-Metagenomics/emg-viral-pipeline/issues/86

**Pull request merged**